docs/openfaas-cloud/secrets.md

```diff
@@ -0,0 +1,116 @@
+OpenFaaS Cloud **secret** objects lets you store and manage sensitive information securely
+
+
+## Using secrets
+
+Secrets in OpenFaaS Cloud will be mounted as data volumes and made accessible to your function
+
+### Creating secrets
+
+To consume a **secret** in a function:
+
+1.\ Create the secret object `secret.yml` using literals or from a file
```

Comment:
The following command will result in a file names `secrets.yml` rather than `secret.yml`